src/schnetpack/transform/neighborlist.py

```diff
@@ -200,6 +202,55 @@ def _build_neighbor_list(self, Z, positions, cell, pbc, cutoff):
         return idx_i, idx_j, offset
 
 
+class ASENeighborListWithSkin(NeighborListTransform):
```

Comment:
add warning and examples to docstring when this should / should not be used. 

In the current form it can only be used with a single system, correct? Could this be also extended to cache the indices of a whole batch, similar to the cached neighborlist? Maybe this could also be a wrapper around a base neighborlist just like the caching?

Reply:
Now its a wrapper. I have only tested it on single system so far. I will look into batch processing asap.

Reply:
There is an idx property in the data that you can use to identify the system.